docs/aerial_autonomy_stacks.md

```diff
@@ -14,17 +14,17 @@ the following autonomy stack table was extracted and adapted.
 
```

Comment:
Let's go ahead and add a comment to warn other possible contributors not to update the tables manually

```suggestion
[//]: # (The following tables are auto generated by CI every week. Please don't edit manually or risk CI breaking.)
```